ups/ap_pipe.table

```diff
@@ -0,0 +1,23 @@
+# For each dependency except python, list it here along with its minimum
+# version number.
+# Very common third-party packages (boost, python, swig, doxygen) 
+# and very low-level LSST packages can be assumed to be recursively
+# included by low-level LSST packages such as utils or daf_base.
+# Any other package whose interface is used should be listed explicitly
+# rather than assuming it will be included recursively.
+setupRequired(boost >= 1.47.0)
+setupRequired(utils >= 4.6.0.0)
+setupRequired(pex_exceptions >= 4.6.0.0)
+setupRequired(ndarray)
+
+setupRequired(obs_decam)
+setupOptional(ap_verify_hits2015)
+setupOptional(ap_verify)
```

Comment:
Definitely shouldn't have `ap_verify` here, because that gives a circular dependency. I don't think `ap_verify_hits2015` is appropriate either.